etc/profile-a-l/chatterino.profile

```diff
@@ -0,0 +1,97 @@
+# Firejail profile for Chatterino
+# Description: Chat client for https://twitch.tv
+# This file is overwritten after every install/update
+# Persistent local customizations
+include chatterino.local
+# Persistent global definitions
+include globals.local
+
+# Also allow access to mpv/vlc, they're usable via streamlink.
+noblacklist ${HOME}/.config/mpv
+noblacklist ${HOME}/.config/pulse
+noblacklist ${HOME}/.config/vlc
+noblacklist ${HOME}/.local/share/chatterino
+noblacklist ${HOME}/.local/share/vlc
+
+# Allow Python for Streamlink integration (blacklisted by disable-interpreters.inc)
+include allow-python3.inc
+
+# Allow Lua for mpv (blacklisted by disable-interpreters.inc)
+include allow-lua.inc
+
+# disable-*.inc includes
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+# Also allow read-only access to mpv/VLC, they're usable via streamlink.
+mkdir ${HOME}/.config/mpv
+mkdir ${HOME}/.config/pulse
+mkdir ${HOME}/.config/vlc
+mkdir ${HOME}/.local/share/chatterino
+mkdir ${HOME}/.local/share/vlc
```

Comment:
To avoid cluttering the home directory with potentially unnecessary paths, just
whitelist the paths of other programs instead of also creating them.

Doing this would probably only help if these programs were never used before
opening them through chatterino (which is unlikely).

Also, AFAIK the configuration file for at least mpv has to be created manually,
in which case creating ~/.config/mpv in the profile wouldn't help anyway.